### ISSUE TITLE
Highlight expected reply by villian in the main board if possible

### ISSUE THREAD

**Issue by Nordlandia** [NiteMoves/tcec-web-public#5]
Additionally it will be a nice to see predicted move accuracy in percentage throughout the game or even for the whole match. 

![2](https://user-images.githubusercontent.com/26534454/41554060-025c5b64-7334-11e8-9ba9-f5c9d3023e14.png)

**Comment by NiteMoves** [comment]
I think displaying the ponder move next to the actual move would be confusing.  Maybe just a popup tooltip.

I agree, having a match/tournament stats table, under the schedule perhaps, would be a nice addition.

@gitlost

**Comment by Nordlandia** [comment]
Displaying ponder move without causing confusion and clutter up things is surely nice thing to have. 

![123](https://user-images.githubusercontent.com/26534454/41743856-4ff3a4f2-75a2-11e8-8589-89f9f0b3611c.png)

**Comment by gitlost** [comment]
Yes putting the ponder move and hit stat as a tooltip on the current move is very doable and looks like a good idea (there's plans to have a nice move listbox and graveyards on it also).

The only thing about the ponder hit stat (assuming you mean just for the current game?) is that it will have to be calculated client-side at the moment but that should be fairly cheap to do...
